src/main/java/uk/gov/hmcts/reform/sscs/services/mapper/CaseDataEventBuilder.java

```diff
@@ -82,7 +87,15 @@
     }
 
     private List<Event> buildPostponedEventsFromMajorStatus(AppealCase appealCase) {
-        MajorStatus latestMajorStatus = getLatestMajorStatusFromAppealCase(appealCase.getMajorStatus());
+        List<MajorStatus> majorStatus18 = appealCase.getMajorStatus().stream()
```

Comment:
Maybe wrap this is ListUtils.emptyIfNull so we don't throw a null pointer at the stream